feature/bgp/policybase/otg_tests/comm_ext_match_action_test/README.md

```diff
@@ -0,0 +1,181 @@
+# RT-7.8: BGP Policy Match and Set Extended Communities Import/Export Policy
+
+## Summary
+
+Configure bgp policy to add communities to routes by matching on the following
+criteria.
+
+* RT-7.8.1 Validate test environment
+* RT-7.8.2 Validate policy to set standard community for all routes using OC release 3.x
+* RT-7.8.3 Validate policy to set ext-community for various match criteria using OC release 3.x
+
+## Testbed type
+
+* https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed
+
+## Procedure
+
+* Testbed configuration - Setup BGP sessions and prefixes.
+  * Generate config for 2 DUT and ATE ports where:
+    * DUT port 1 to ATE port 1.
+    * DUT port 2 to ATE port 2.
+  * Configure ATE port 1 with an external type BGP session to DUT port 1.
+    * Advertise ipv4 and ipv6 prefixes to DUT port 1 using the following communities:
+      * prefix-set-1 with 2 ipv6 and 2 ipv4 routes without communities.
+      * prefix-set-2 with 2 ipv6 and 2 ipv4 routes with communities `[5:5, 6:6 ]`.
+      * prefix-set-3 with 2 ipv6 and 2 ipv4 routes with extended communities `[50:500000, 60:600000]`.
```

Comment:
this are NOT extended community.

Reply:
ok, removed type 1 community test since we do not need it at this time.